clean.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+REM Copyright (c) .NET Foundation and contributors. All rights reserved.
+REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if "%~1"=="" (
+  @call run.cmd clean -?
+) else (

```

Comment:
It is true that the check is for that, but you are not returning the exit code of this call. 
You could do:
```
if "%~1"=="" (  
   @call run.cmd clean -?  
   @exit /b 1
)
@call run.cmd clean %*  
@exit /b %ERRORLEVEL%  

```